src/openforms/forms/api/serializers.py

```diff
@@ -153,6 +153,14 @@ class FormSerializer(serializers.ModelSerializer):
         default=list,
     )
     login_options = LoginOptionsReadOnlyField()
+    auto_login_authentication_backend = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        help_text=_(
+            "The authentication backend to which the user will be automatically "
+            "redirected upon starting the form. The chosen backend must be present in `authentication_backends`"
+        ),
```

Comment:
```suggestion
        help_text=_(
            "The authentication backend to which the user will be automatically "
            "redirected upon starting the form. The chosen backend must be present in "
            "`authentication_backends`."
        ),
```
Please also keep the line-lengths in mind, black is configured for 88 columns.